lib/data_migrate/database_tasks.rb

```diff
@@ -179,7 +179,9 @@ def self.migrate_with_data
 
       ActiveRecord::Migration.verbose = ENV["VERBOSE"] ? ENV["VERBOSE"] == "true" : true
 
-      schema_mapped_versions = if DataMigrate::RailsHelper.rails_version_equal_to_or_higher_than_7_1
+      # 7.2 removes the param for db_configs_with_versions in https://github.com/rails/rails/commit/9572fcb4a0bd5396436689a6a42613886871cd81
+      # 7.1 stable backported the change in https://github.com/rails/rails/commit/c53ec4b60980036b43528829d4b0b7457f759224
+      schema_mapped_versions = if Gem::Dependency.new("railties", ">= 7.1.4").match?("railties", Gem.loaded_specs["railties"].version, true)
```

Comment:
and there still no support for rails < v7, because the _db_configs_with_versions_ wasn't implemented before v7

Reply:
Right right, I see https://github.com/rails/rails/commit/45eb0f3beccc966689ddf76d1d3c0618852f8af6 is the commit that implemented it... Weirdly enough it didn't raise any issue when manually testing data-migrate with a rails 6.1 codebase.

I will dig into that further.

Reply:
@fobiasmog is your team using any of the features coming with data-migrate v11 on rails 6.1?